test/py/libvfio_user.py

```diff
@@ -899,22 +1038,76 @@ def vfu_setup_device_dma(ctx, register_cb=None, unregister_cb=None):
                                                 vfu_dma_unregister_cb_t))
 
 
+# TODO group all callbacks in some class?
```

Comment:
clean up?

Reply:
Not sure it's worth fixing, I'm removing the TODO.